CRM/Activity/BAO/Activity.php

```diff
@@ -1088,13 +1084,11 @@ public static function sendEmail(
       }
 
       $tokenSubject = $subject;
-      $tokenText = in_array($values['preferred_mail_format'], ['Both', 'Text'], TRUE) ? $text : '';
```

Comment:
removing this was how it came up - might as well make our grepping lives easier even if we never call this